ts/Grid/Pro/CreditsMore.ts

```diff
@@ -0,0 +1,164 @@
+/* *
+ *
+ *  Grid Credits class
+ *
+ *  (c) 2020-2024 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Dawid Dragula
+ *  - Sebastian Bochan
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type { CreditsOptions } from '../Core/Options';
+import type Table from '../Core/Table/Table';
+
+import Credits from '../Core/Credits.js';
+import U from '../../Core/Utilities.js';
+
+const {
+    addEvent,
+    merge
+} = U;
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * Represents a credits in the data grid.
+ */
+class CreditsMore extends Credits {
```

Comment:
Can't we have it all in the namespace? Do we need to extend it?

Reply:
Do you mean Composition instead of "extend"? If so, I'm not convinced, because we can overwrite methods very easily in this solution. Otherwise  we create a lot of code/wraps, so the bundle will be bigger.